po/POTFILES.in

```diff
@@ -1,5 +1,26 @@
 # This is a list of files which contain translatable strings.
 # This file was autogenerated by cmake.
```

Comment:
See this line! We need a pattern CMakeList.txt, which searchs for this files.

Reply:
I'll remove this file. I'll change the line in CMakeList.txt. But the latter is just cosmetic as I have already included *.py.